src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.security.SecureRandom;
+
 public class ColorSupplier {
+    private static final int COLOR_LENGTH = Colors.values().length;
+    private static final SecureRandom RANDOM = new SecureRandom();
+
     public String getRandomColor() {
-        return null;
+        return Colors.values()[RANDOM.nextInt(COLOR_LENGTH)].name();
```

Comment:
Better to extract RANDOM.nextInt(COLOR_LENGTH) in local variable to make your code more readable.